automated/linux/perf/perf.sh

```diff
@@ -11,19 +11,21 @@ TEST_FAIL_LOG="${OUTPUT}/test_fail_log.txt"
 TEST_SKIP_LOG="${OUTPUT}/test_skip_log.txt"
 export RESULT_FILE
 SKIP_INSTALL="false"
+RUN_TESTS_IN_PARALLEL="false"
 # List of test cases
 TEST="record report stat test"
 # PERF version
 PERF_VERSION="$(uname -r | cut -d . -f 1-2)"
 
 usage() {
-    echo "Usage: $0 [-s <true|false>]" 1>&2
+    echo "Usage: $0 [-s <true|false>] [-p <true|false>]" 1>&2
```

Comment:
Can we new line this? so a echo for each

Reply:
Yes, the new version has changed to:
```
Usage: ./perf.sh [options...]
 [-s <true|false>]        Skip to install perf
 [-p <true|false>]        Run tests in parallel
```